applications/advanced-concepts/tags-and-placeholders.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Tags and placeholders in machine translation
+---
+
+# Tags and placeholders
+
+**Tags and placeholders** are pieces of non-translatable code. While tags format and structure texts, placeholders are variables for dynamic content.
+
+## Challenges
+
+- The machine translation can misplace, corrupt or ignore tags and placeholders.
+
+In a phrase like "Susan's car", the genitive apostrophe can appear as a tag. When translated into Spanish, the appearance of the tag in the output introduces an error.
```

Comment:
I do not understand this example, to be honest.

I'd go for something more basic and concrete.

> Original:
> > English: There are {pages} pages in this book.
>
> Expected translation:
> > Spanish: Hay {pages} páginas en este libro.
>
> Translation with a corrupted placeholder:
> > Spanish: Hay { páginas } páginas en este libro.
>
> Translation with a dropped placeholder:
> > Spanish: Hay páginas en este libro.

But not sure it's necessary.